kvm/requests.yaml

```diff
@@ -9,3 +9,6 @@ releases:
         - name: calico
           version: ">= 3.15.1"
           issue: https://github.com/giantswarm/giantswarm/issues/11307
+      except:
+        - releaseVersion: 12.2.0
+          reason: kvm-operator work needed
```

Comment:
```suggestion
          issue: https://github.com/giantswarm/giantswarm/issues/11307
          except:
            - releaseVersion: 12.2.0
              reason: kvm-operator work needed
```

Reply:
Thanks for the tip, saw it late, fixed it already.